src/python/docs/docstrings/Dart.txt

```diff
@@ -7,9 +7,9 @@
         <https://arxiv.org/abs/1505.01866>`_ is an
         ensemble method of boosted regression trees. The Dropouts meet
         Multiple Additive Regression
-	Trees (DART) employs dropouts in MART and overcomes the issues of over-
```

Comment:
>	 [](start = 0, length = 1)

This has been fixed in PR 369, right? Why its coming up again?

Reply:
Or do you need to rebase again?

---
In reply to: [349970007](https://github.com/microsoft/NimbusML/pull/372#discussion_r349970007) [](ancestors = 349970007)

Reply:
@ganik Yes, this was fixed in PR 369 into **master**. This is merging into temp/docs branch, which doesn't have this change. temp/docs is where I want the change in order to auto generate the API docs.